pyretailscience/plots/histogram.py

```
@@ -0,0 +1,266 @@
+    Returns:
+        pd.DataFrame: The modified dataframe with the clipping or filling applied.
+    """
+    if range_lower is not None or range_upper is not None:
+        if range_method == "clip":
+            return df.assign(**{col: df[col].clip(lower=range_lower, upper=range_upper) for col in value_col})
+        return df.assign(
+            **{
+                col: df[col].apply(
+                    lambda x: np.nan
+                    if (range_lower is not None and x < range_lower) or (range_upper is not None and x > range_upper)
+                    else x
```

Ruff

Trailing comma missing

Add trailing comma

(COM812)